CHANGELOG.md

```diff
@@ -57,12 +59,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Fixed typos in ind_ variable names in `KPP/carbon/carbon_Funcs.F90`
 - Fixed typo in GCHP operational run script for Harvard Cannon to properly retrieve the run duration string
 - Fixed bug in ObsPack to include instantaneously-sampled data whose  timestamps are within 1/2 of a model timestep of the end of the day 
+- Fixed indexing error in routine `Grav_Settling` in module `GeosCore/sulfate_mod.F90`
```

Comment:
I recommend expanding this to "Fixed indexing error in routine Grav_Settling in module Geos/Core/sulfate_mod.F90 which caused incorrect dry deposition diagnostics for some species." This assumes that error is what caused the original issue.